hudi-common/src/main/java/org/apache/hudi/common/fs/HoodieWrapperFileSystem.java

```diff
@@ -90,11 +94,18 @@ public static void setMetricsRegistry(Registry registry, Registry registryMeta)
     R get() throws IOException;
   }
 
+  @FunctionalInterface
+  public interface CheckedIntFunction extends CheckedFunction<Integer> {
```

Comment:
What is this used for ?

Reply:
This is used for one of the executeFuncWithTimeAndByteMetrics() function as the type of argument.

This refers to a API function which returns an integer as the number of bytes written.

Reply:
Okay, thanks for clarifying